tests/automation/types/testing.ts

```diff
@@ -11,10 +13,52 @@ export type Group = {
   userThree: User;
 };
 
-export type Strategy = 'data-testid' | 'class' | ':has-text';
+export type ConversationType = '1:1' | 'group' | 'community' | 'note-to-self';
+
+export type DMTimeOption =
+  | 'input-10-seconds'
```

Comment:
That's the radio button element, otherwise I can't determine if it's 'checked' or not. The data test id is not on the radio button, so it errors when I try to check the attribute for checked